src/containers/navbar/navbar.component.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { Route, Link, BrowserRouter as Router } from "react-router-dom";
+import { useSelector } from "react-redux";
+
+const Navbar = (props) => {
+  console.log(props.navigation);
+  return (
+    <div className="border-grayy align-middle border-b-2 flex justify-between pt-3 pb-1 text-l bg-bluey">
+      <div>
+        <i className="pl-2  material-icons color-grayy">menu</i>
+      </div>
+      <div className="font-mono text-grayy  text-center">Undefined</div>
```

Comment:
Can we call all the Undefined something else please it gives me PTSD from bugs lol